Sources/OTelTesting/RecordingLogHandler.swift

```diff
@@ -0,0 +1,41 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift OTel open source project
+//
+// Copyright (c) 2024 Moritz Lang and the Swift OTel project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+import Logging
```

Comment:
```suggestion

import Logging
```

Reply:
Thanks. Good point, I created an issue for adding this: https://github.com/slashmo/swift-otel/issues/87